content/post/2017-05-22-using-asdf-vm.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Using ASDF-vm
+subtitle: Solve All of Your Version Problems
+date: 2017-05-21
+author: Connor Rigby
+draft: false
+tags: ["nerves"]
+---
+
+Recently there has been a number of breaking changes in the Erlang/Elixir world that have caused Nerves-based projects to fail to compile, causing a lot of confusion in slack. Follow along as we solve all your version inconsistencies with ASDF-vm.
```

Comment:
yeah i suppose "number of" is a little bit of an exaggeration. 
but even then in reguards to #1 i saw a ton of breakage due to the `Version` module when using 1.4.4 and OTP < 20.